common/static/common/js/components/views/paginated_view.js

```diff
@@ -0,0 +1,43 @@
+;(function(define) {
+    'use strict';
+    define([
+        'backbone',
+        'underscore',
+        'common/js/components/views/paging_header',
+        'common/js/components/views/paging_footer',
+        'common/js/components/views/list',
+        'text!common/templates/components/paginated-view.underscore'
+    ], function (Backbone, _, PagingHeader, PagingFooter, ListView, paginatedViewTemplate) {
+        var PaginatedView = Backbone.View.extend({
+            initialize: function (options) {
+                var ItemListView = ListView.extend({
```

Comment:
@peter-fogg This component is meant to be generic, right? I don't think we should be statically wiring in `tagName` and `className`, and we also shouldn't assume a `this.cardView` property, nor the fact that it takes a router.